spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/event/AssetDeleted.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.spi.event;
+
+import java.util.Objects;
+
+public class AssetDeleted extends Event {
```

Comment:
maybe a line of documentation?

Reply:
@ndr-brt the name of the class is pretty self-explanatory, I find it a bit difficult to write something that doesn't sound totally useless :)

Reply:
How about what "deleted" means and what it entails, e.g. is "deletion" a marker and the Asset entity still in the backing store?

Reply:
@jimmarino ok, got it, I would avoid the detail about the logical/phisical deletion, as it is an implementation delay of the store layer (I guess). 
I reported that, after the event emission, the asset won't be available anymore.